File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] arr = new int[nums.length];
        int leftP = 0;
        int rightP = n;
        for(int i=0; i <nums.length ; i++ ){
            if(i%2==0){
                arr[i]= nums[leftP];
                leftP++;
            }
            else{
                arr[i]=nums[rightP];
                rightP++;
            }
        }
        return arr;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> ans(nums.size());
        int i = 0;
        int j = n;
        int k = 0;
        while(i < n){
            ans[k] = nums[i];
            ans[k+1] = nums[i + n];
            i++;
            j++;
            k+=2;
        }
        return ans;
        
    }
};